index.html

```diff
@@ -12,11 +12,34 @@
 
 <body>
     <header>
-        <div class="mobile">
-            <a href="#"> Ikpro </a>
-            <nav>
-                <a href="#"><i class="fa-solid fa-bars"></i></a>
-            </nav>
+        <div >
+            <div class="mobile flex1">
+                <a href="#"> Ikpro </a>
+                <nav>
+                    <a href="#" id="hamburgerButton"><i class="fa-solid fa-bars"></i></a>
+                </nav>
+            </div>
+            
+            <ul id="contactList" class="disappear ">
+                <li class="mobile flex1">
+                    <a href="#">  </a>
+                    <nav>
+                        <a href="#" id="hamburgerButton2"><h3>&times;</h3></a>
+                    </nav>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#Projects" class="options"> <h3 class="mobile-menu">Portfolio</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#About" class="options"> <h3 class="mobile-menu">About</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#contact" class="options"> <h3 class="mobile-menu">Contact</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
```

Comment:
- [ ] the arrows next to links doesn't have the same colors as the ones in the Figma design so , please try to export the arrow in the Figma and use them in your menu in order to stick to the design 

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/167719639-ee0a2888-7e76-4cb6-b581-fc7dbf343317.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/167719672-dd48e6e0-c2d9-47dc-8d41-fed392ad8e32.png)